house/house.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+const xml2js = require('xml2js')
+
+BillFetcher.prototype.FetchBills = function (date, cb) {
```

Comment:
cb is short for callback, which is a function that will execute when an asynchronous request is done... 


https://en.wikipedia.org/wiki/Callback_(computer_programming)